packages/pds/src/api/com/atproto/server/createAccount.ts

```diff
@@ -52,28 +53,42 @@ export default function (server: Server, ctx: AppContext) {
 
       // determine the did & any plc ops we need to send
       // if the provided did document is poorly setup, we throw
-      const { did, plcOp } = await getDidAndPlcOp(ctx, handle, input.body)
+      const signingKey = await Secp256k1Keypair.create({ exportable: true })
+      const { did, plcOp } = await getDidAndPlcOp(
+        ctx,
+        handle,
+        input.body,
+        signingKey,
+      )
+      const commit = await ctx.actorStore.create(
```

Comment:
I think I may have mentioned it on the account manager PR, but it could be nice to clean this new repo/keypair up in the case that registration fails in a subsequent step.

Reply:
sweet handled it in the account manager PR since the route is a bit simpler there 👌